src/primitives/AllPrimitiveTypes.java

```diff
@@ -0,0 +1,27 @@
+package primitives;
+
+public class AllPrimitiveTypes {
+    public static void main(String[] args) {
+
+        int addNumber = 75;
+        short addShort = 82;
+        long addLong = 84637;
+
+        double addDouble = 65.889;
+        float addFloat = 364.75f;
+
+        char addChar = 'x';
+        boolean addBoolean = true;
+
+        byte addByte = 122;
```

Comment:
you have incorrect variables' names. not 'addNumber'. 'addNumber' is correct name for the method. if method calls 'addNumber' it means that this method probably adds number to another number. 
intNum completely clear name in this case. so change names to all the variables.

additionally provide all types from littlest to highest. so boolean should be first one, it takes in memory only one bit (https://www.w3schools.com/java/java_data_types.asp)